x-pack/plugins/security_solution/cypress/integration/timeline_templates_export.spec.ts

```diff
@@ -5,44 +5,41 @@
  */
 
 import { exportTimeline } from '../tasks/timelines';
-import { esArchiverLoad, esArchiverUnload } from '../tasks/es_archiver';
 import { loginAndWaitForPageWithoutDateRange } from '../tasks/login';
-import { timeline as timelineTemplate } from '../objects/timeline';
+import {
+  expectedExportedTimelineTemplate,
+  timeline as timelineTemplate,
+} from '../objects/timeline';
 
 import { TIMELINE_TEMPLATES_URL } from '../urls/navigation';
-import { addNameToTimeline, closeTimeline, createNewTimelineTemplate } from '../tasks/timeline';
+import {
+  createTimelineTemplate,
+  deleteTimeline as deleteTimelineTemplate,
+} from '../tasks/api_calls/timelines';
+let template = '';
 
 describe('Export timelines', () => {
-  before(() => {
-    esArchiverLoad('timeline');
-    cy.intercept('PATCH', '/api/timeline').as('timeline');
-    cy.intercept('POST', '/api/timeline/_export?file_name=timelines_export.ndjson').as('export');
+  before(async () => {
+    cy.intercept('POST', '**api/timeline/_export?file_name=timelines_export.ndjson*').as('export');
+    template = await createTimelineTemplate(timelineTemplate);
   });
 
-  after(() => {
-    esArchiverUnload('timeline');
+  after(async () => {
+    const templateId = JSON.parse(JSON.stringify(template)).savedObjectId;
+    deleteTimelineTemplate(templateId);
   });
 
   it('Exports a custom timeline template', () => {
     loginAndWaitForPageWithoutDateRange(TIMELINE_TEMPLATES_URL);
-    createNewTimelineTemplate();
-    addNameToTimeline(timelineTemplate.title);
-    closeTimeline();
 
-    cy.wait('@timeline').then(({ response }) => {
-      const {
-        savedObjectId: timelineId,
-        templateTimelineId,
-      } = response!.body.data.persistTimeline.timeline;
+    const jsonTemplate = JSON.parse(JSON.stringify(template));
```

Comment:
`JSON.parse` is opposite to `JSON.stringify`, so seems you don't need any of these as `template` is the same as `JSON.parse(JSON.stringify(template))`